### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by unlight** [cypress-io/cypress#74]
Seems that windows platform is not supported.

**Comment by brian-mann** [comment]
Correct, Windows is not **yet** supported. There is only a handful of early adopters who have required Windows so it has not been a priority.

However, it will be *fairly* easy to add it, and we plan on integrating Windows by the time we are publicly live.

Are you using Windows in CI as well? Or just locally while you develop? Any additional info about your organization / its size would be helpful to prioritize Windows support.

**Comment by buzinas** [comment]
:+1:

**Comment by jblaketc** [comment]
I'm in a company of 7. We all use windows for local dev.

**Comment by jennifer-mann** [comment]
Error message updated in [0.15.3](https://github.com/cypress-io/cypress/wiki/changelog#0153-04102016) when desktop app opened in Windows:

```text
Sorry, Windows is not yet supported. We currently support OSX and Linux.

You can use Cypress if you install a Linux VM using something like VirtualBox.

You can track this issue here:

https://github.com/cypress-io/cypress/issues/74
```

**Comment by bradygaster** [comment]
Is there any ETA on the resolution of this issue?

**Comment by jennifer-mann** [comment]
Windows will be supported prior to our public release, there is not a specific date set for this.

**Comment by adest** [comment]
+1

In our company we also have a lot of windows system for local dev.
Can't wait to be able to test your testing framework on Windows.

**Comment by kurtommy** [comment]
+1

**Comment by jennifer-mann** [comment]
@jayfresh Check the open issue for a Cypress Docker image here: #165

**Comment by lennartquerter** [comment]
+1

Our company only uses windows + windows servers for CI. Atm i have to run a VM to use cypress and we can not do any testing on CI.

**Comment by abadi199** [comment]
+1
would love to be able to run cypress on my windows dev machine.

**Comment by mpseidel** [comment]
+1
Primarily on the dev machine but support for running CI on Windows would be great, too so customers of Visual Studio Team Services can run it.

**Comment by erlendr** [comment]
+1

**Comment by jzhang52** [comment]
+1 
Our dev machines are on Windows, but we are thinking about setting up some Docker image on Linux for CI.

**Comment by cwortel** [comment]
+1 most dev machines at ING Netherlands are

**Comment by jtwite** [comment]
+1 Half of our UI devs use windows and all of our QA.

**Comment by SuperDrew** [comment]
:+1: 
Windows based shop, dev, QA and CI all windows.

**Comment by Mair** [comment]
+1

**Comment by montag** [comment]
I was excited to try cypress but I'm on win7.

**Comment by jhanink** [comment]
@brian-mann @jennifer-mann - in regard to `fairly easy to add` mentioned on top, in orders of magnitude, is that weeks or months, and is a live public release in 2016 a possibility, or is it not on near horizon?

**Comment by paulblyth** [comment]
Also voicing support for Windows... err... support. I was excited to show colleagues the tool but we're running a mix of Mac/Win with the majority being Windows users so that's going to be tricky

**Comment by victorwpbastos** [comment]
+1

**Comment by adam-janis** [comment]
+1

**Comment by activebiz** [comment]
+1

**Comment by secelite** [comment]
+1

**Comment by gaboma19** [comment]
+1

**Comment by oneillci** [comment]
+1 We are a Windows only shop for both dev and QA. Would love to try Cypress and also integrate with our CI - either VSTS (Visual Studio Team Services), Visual Studio Team Build or Jenkins

**Comment by cdoucet6302** [comment]
+1

**Comment by about0** [comment]
Windows, please!

**Comment by CheungJ** [comment]
+1

I first saw Cypress at JS Monthly London and was so disappointed to hear there was currently no Windows support.

Working in a company where we use Windows. Our tests are written using Protractor and I'm looking forward to attempting a switch to using Cypress. It just looks so user friendly and easy for debugging! ☺️

**Comment by MarioTupy** [comment]
Windows ! C mon guys, you can do this !

**Comment by caerie4** [comment]
+1

**Comment by zobidafly** [comment]
Cypress looks very promising, unfortunately no support for Windows... So for me, it's just an empty promise as long as there's no Windows support.
too bad! Meanwhile, Jasmine, Require, Marionette etc within TypeScript will be my day-to-day job...
Another reason why Windows is evilish...
![image](https://user-images.githubusercontent.com/4248052/27833198-046d92bc-60d2-11e7-8ca8-19922dbbf69d.png)

_sigh_

**Comment by nartamonov** [comment]
+1 We have all dev machines with Windows 7.

**Comment by gmschouten** [comment]
+1 (team of 5)

**Comment by Strajk** [comment]
We eventually solved this issue by buying Macbooks for whole team... #quickfix :D

**Comment by tomexx** [comment]
Any update on this? Or is there any way we can help?

**Comment by AshMcConnell** [comment]
We would really appreciate a windows version too :+1:

**Comment by Roobyx** [comment]
I was really disappointed to find out there is no Windows support. For the past 6 years of my career I've nearly always worked in a mostly Windows dev evironments in small and big companies. 
I will try it out on a Linux VM and I would hope all would work out fine !

+1 for the Windows port !

**Comment by petermouton** [comment]
I'm working for the Dutch government, whete Windows is standard. So please provide a Windows version! Mac is a niche market compared to Windows!

**Comment by arvinlaqui** [comment]
Please have a windows version

**Comment by blackhill** [comment]
+1 for havinf a windows version and support in visual studio online CI

**Comment by zedL** [comment]
windows +1

**Comment by mmiszy** [comment]
> we plan on integrating Windows by the time we are publicly live.

@brian-mann as far as I understand cypress is now publicly live. Is there an ETA for Windows support?

**Comment by brian-mann** [comment]
We are not publicly live. Windows is something we are looking at investing developer time into supporting but it will happen after being live. You'll see our 0.20.0 + 1.0.0 release soon.

**Comment by frattaro** [comment]
I didn't want to wait. Installed VirtualBox, Ubuntu, java8, Eclipse and the TFS plugin for it, nodejs-legacy, npm, git, cypress-cli, cypress and started coding.

Outside of overcoming the learning curve of all of the above (never used any of them), I ran into the following issues along the way:
VirtualBox over Cisco's AnyConnect VPN, no internet access in Ubuntu instance
apt-get install nodejs doesn't work, needs nodejs-legacy
npm registry blocked by corporate firewall
git is an unlisted dependency of cypress-cli

It was very late in the process I saw a small mention of a TFS plugin for IntelliJ, which I hear is "better".

Maybe I'll write a blog or something about it.

**Comment by lifenstein** [comment]
+1 for Windows support. CI Build server runs Linux, devs on Macs and Windows.

**Comment by NeverwinterMoon** [comment]
Our CI has both, Linux and Windows machines, but our devs are mostly on Windows (dictated by the company policy for some weird reason), so it would be nice to be able to run Cypress on Win indeed!

**Comment by bahmutov** [comment]
We got Cypress running on Win10 (inside VirtualBox), and we need a few more tweaks to make this production ready. Here is a screenshot, wish there was a screenshot of the team standing around the laptop and watching Cy run on Windows 😀

<img width="1021" alt="screen shot 2017-09-20 at 2 09 01 pm" src="https://user-images.githubusercontent.com/2212006/30660077-b1255792-9e0d-11e7-98bb-48dd93c44b32.png">

**Comment by AshMcConnell** [comment]
Great news, thanks Gleb and team!

**Comment by teelicht** [comment]
+1 ... would be very interested to see this working on win 10. probably relying on the now available linux subsytem in windows 10 pro could speed things up for you.

**Comment by bahmutov** [comment]
I am fighting Electron crashes on Windows :( But here is a pic of Chrome browser controlled by Cypress running tests
![virtualbox_win10_25_09_2017_19_28_35](https://user-images.githubusercontent.com/2212006/30863208-796a11ea-a29e-11e7-8e30-21e6fc4d5a66.png)

Ha, I just noticed both forward and backward slashes in the url 😀

**Comment by JIGD** [comment]
+1

**Comment by wickdninja** [comment]
👍 

using windows dev machines and running Windows in CI via Jenkins

**Comment by benviau** [comment]
I'm working for an international company. More than 50 devs in this building alone, all using Windows for local development. Requiring Macs is a dealbreaker for us, probably as well as 90% of the market. Looking forward to that Windows support!

**Comment by lanceweatherby** [comment]
Windows in on our roadmap and currently in development. https://docs.cypress.io/guides/references/roadmap.html#Upcoming-Features

**Comment by brian-mann** [comment]
Fixed in [`1.0.0`](https://docs.cypress.io/guides/references/changelog.html#1-0-0).

**Comment by mateatslc** [comment]
_still looking for the 'holy shit batman' emoji to celebrate_
